twitter.html

```diff
@@ -0,0 +1,275 @@
74pseXYXLg1IBUwuOdAWU5wRnNOnBGu3LalFaWlvFHHKQkihyE+YHY7H55FJlwyyY9Omff2qZws0q8Q6eIXKu06rn3JxTyVolF0zfQARkV4VrtQ3KOcYbuK+NZCwRBWe/bRE/3LSpwf8ACR5EI/1EAg/RTWioM0l/a9aSXHDlq0QLeHdZI/8ABv7U+P8A0LLwxuLGN++9dIC0oVerfsKCRhuX0GKmabGZLuJcZy1apOkTiraRpfCmjW9vbxuVDSkfEQNqbEiU4qm0gMsKKvZRV/bRMw3Fea25M9akvCRbRgEEDpVnCuw2qPbQkbYqwjTAq2NUQyME4I6HFV2o5Iq4KbVAv4wVO1NNcEg6Yq3sQkDZpM4nsoxbyPyLnGSQoyf1p6niYZ2pR4tDfcZQOuKhDkjRPsTOpAGTAAHJnamr7MrS3u9WuYbyAywyRhCSuVDbkA+mQDg+opRD/F69KfPspvIIbi5jlPmkli5QFJP4gDkdMEgb+vzrfLw8v9NXC8o5ck423NeEUQrmuDGKyFiQi1A4k0kazodzYluUuAUb8rA5BqxTpXGoO8enXLx/EImI+lFOjkraR+aNStpbK+mgnXEkbFX9zVnwlb/eNRVuvKN/arviTh6e4tV1KFQzsTzoBjIHpX32c2uZLqRl6MF3qssqljtDxwuGbVjGmvwWJaKOGSZ1GCQMAUa249nt2HPpBK92JI2+lG1fxdOs3e1iVpseXK5qgkh4gmsYLxdUlMjOfGtkRF8NcZGMjf0zv22qOJJlc0pIeNP+0DRrkKtwHtZcdGwy/UUz2N/BexLJbyJIh7qciseXT7uK3hnvZxc3LuQ0TW2SidiWVf19qd+CiIC9tycig5xTSlToWKbjY4zXEUEbPKwVB1JPSlLVuPdDtnMcTTXcg6rAmR9TtXfGcsjxx20aFhJ1UNjNZ/Npl3LaTyWlwbOdCojhW3wXGfN5mHXHvXKVujpRajaLa+44upzm30tlTOwIJP8ALFVH8W/jMjwzwtEx/DjrQYouIIreWU6nKWVwI4HCtzr33x/v6Ve2UE12kU92g8VB8QXFDJUQ4tpGW3SeFcTJ0CtitD+xlVa81VWRSBHCwyOhBbH86VJ9JnvdcvIox5Ul83tmtY+zrhuLRtKe55ua4ujlm9FBOBVpTWtGb+t3tXBnIrk0VxihGoDBVNezr4ttLGMEshG9DSirQsPgoSWs33NbaNmVXd1b127VU8NaW2l397byb5lDqT1IYAinDVLVuZsKWjlIJ5Tgqw7iqS5intNTieeYSCRcKceYBfX61N84bdlOmWstsJ0CMM7V7HpbcgUOQvtR7Zudlwe1W0SZWnikxZOhfm0lIkZlyW9TUTRoXjvAqscnr70w6kyxWrsxwAKrdCVprrn5SN9qDSsKfzbBcSW/jXMUMx8wGcjtUOHROUBss2e5berjiiFllSZRllO+3aj6awmtFIwQevtRr6OT+SpGjRyfG7NjsTQruBYIyoGAB0piaLkBxVBrLBEIJpZpBi+ibodo54lv7iRcQTIVB9cDr9Qa0jQhy6RbdRlebHzNKWj6dLFb5VvEe5YhPSPmp5iiWCCOJeiKFH6UyduyWZpRUUfE5FBYUVtqEDk0WZ0eo1FXpQkIAoiMKBx2VDqVboaotfsQlqs/OWKOOo6A7f2pgWgajb/erGeAYy6EDPY9v3otWgwm4spNMkOBk9KZLXzJSppcnMoyCpI3B6imnT3HhZNJA05PCLrkBaychcspB5c9d6oNH1uW1uCHsuSM583NnB99v70zXlxBGpM8qIp/McVV81i3MQxIbbmCZFM120dB/NNEDXuJZHnjght/Ez8bHp+3WrnQYmW0BdeUsebHpVazacJAd1C9WZcCrmxvbWRQIp4mPYKwJrl7YZ8jSQe6A5P0pM1g+JcJESQJHCnHucU338oRM+opRYC616yiH/fVv0U839KWXWdB0mxj0/So7DBV2lZdlLAAAVNJ7V053rhmBG25p/DI25dYN6Dy710xcHeuOYk9MUDjlFB7miDKmgBmHajK5ONqAWTI9xk12cYoSHai9qdCMTL65Wz4iuLcjkVyrp7kjJ/fNX2mXPMhU9u1U3Elktzqkp351jQg/Wo+k3r21wILrZxsG/MKk+M3RW0EWeoaPYXdwbiWBTI3U7717b6PpyjH3eEdvKxU1ZIqzbgjepC6VHKMvn5Cni2dvqUz6Lpzf9PCT6t5jUnTdJ020Pix28SzHpJyDIFWn8Piij8o6VX38n3dGYnCLuTXO0DfflkTW71UQ74VR1rjhfThJjV5GB8RCIQOy9z8z/8AdaTNXv7jVHdIARApwzD8Rp+4QBThnTk/LDj9zQiui5vmNFryliSKG0ZXo2DRfE5M7ZzQXYscnb2p3RlBk8h83X1oZ3PSu5QCQetfdulKEiq7flo0bZO4xXkbDl6V3kV1HWdXV0lnZz3UgJWJCxA747Vkuv8A2g8TQXbRRTW8CtuoihBwPTLZzTXx1xFHFD/CbbDyysolb8g5gcfOs81K3+96rAXOAowfer40krZ2uyHbg2/1HU7U3GrXDTzuSAzADC9hsBV1eWK3CZPxDoRVZw6nIUVRgdgKaUj9RWWT2bZtXykkVmm6jJYsI7sMyjo4Hb3pht9Zs3AZbmLlA7sKgz2aSb8tQJ9KjlPmUH5iirQkqkXV5xHpkCHnuUY/lQ8xP6ClPVry716XwUjaCyB3UnzSfP0qys9FiQk8oG/pU8WkcY8qii22GOsfBdls47a1MaKFGO1Ldzx3xBov+Ugt7E20ICxs8TE498MN6cdWTCNttikDXLb7xDIoPK3y60cTSl07JHaI8cBcWXvETzwajFAsqxCVHhBUEE4wQSfbf3ptYCsf4H1A6TrzSqglHgNHyc3LnGP7Vo//ABLYJym85rZWGRK28f8A7Dp+uKtOHeGTwteXc19jNcW9zDdQiW2mjljbo8bBgf1FdnrUqOP/2Q==" alt="user image"></label>
```

Comment:
Oh my gosh this image link. Was this pulled directly from a website? Could it have been downloaded and referenced to?

Reply:
You're right. It's not looking neat and I will solve this.